python/src/aiconfig/editor/server/server.py

```diff
@@ -34,7 +34,7 @@ class EditServerConfig(core_utils.Record):
     aiconfig_path: Optional[str] = None
     log_level: str | int = "INFO"
     server_mode: str
-    parsers_module_path: Optional[str] = None
+    parsers_module_path: str = "python/src/aiconfig/editor/default_user_model_parsers_plugin.py"
```

Comment:
I don't think this default makes much sense. It should be looking for a file in the directory where the server was started from, no? Otherwise this local path is just for our own local development...

Reply:
Right, good catch.

Reply:
But the problem is if you start the server from an arbitrary directory, then we cant include the default file, which means if the user forgets to put their module in their directory, they'll get a warning on server start. This is not the end of the world, but it seems weird that the default behavior is to start up with a warning that we can prevent.

Reply:
So I made it look in cwd for `aiconfig_model_registry.py`, which will not exist unless the user puts it there (the cwd could be outside the repo, or maybe the user doesn't even have a clone). 

This means the default server behavior is to print a warning that the file doesn't exist. I think this is fine for now. On the `load_model_parser_module` endpoint, it's stricter and logs a bad file as an error.